UPGRADE-3.x.md

```diff
@@ -1,6 +1,14 @@
 UPGRADE 3.x
 ===========
 
+UPGRADE FROM 3.2 to 3.3
+=======================
+
+### AMQP
+
+You might face some issues (though we tried to keep everything BC) if you are extended AMQPBackend, AMQPBackendDispatcher or AMQPMessageIterator classes 
```

Comment:
"are extending" or "have extended", please pick one. Also, please quote class names with backticks and end sentences with dots.